src/uct/ib/base/ib_log.c

```diff
@@ -177,19 +180,20 @@ static void uct_ib_dump_send_wr(uct_ib_iface_t *iface, struct ibv_qp *qp,
     uct_ib_dump_wr(qp, op, wr, s, ends - s);
     s += strlen(s);
 
-    uct_ib_log_dump_sg_list(iface, UCT_AM_TRACE_TYPE_SEND, wr->sg_list, wr->num_sge,
+    uct_ib_log_dump_sg_list(iface, UCT_AM_TRACE_TYPE_SEND, wr->sg_list,
+                            ucs_min(wr->num_sge, max_log_sge),
```

Comment:
uct_ib_log_dump_sg_list already has access to max_log_sge from ib_iface->ib_device